app/presenters/defect_event_presenter.rb

```diff
@@ -3,7 +3,7 @@ def initialize(event)
     @event = event
   end
 
-  def description
+  def description # rubocop:disable Metrics/CyclomaticComplexity
```

Comment:
I'm wondering what we can do about this given that this rubocop lint is the reason I extracted this class in the first place.